Commbox.json

```diff
@@ -1592,6 +1608,463 @@
         }
       }
     },
+    "/users/objects/{USER_IDENTITY_ID}": {
```

Comment:
This is the wrong path.

Should follow REST conventions - /users/{USER_ID}/identities/{USER_IDENTITY_ID}/objects/